### PR TITLE
Added first and last (#294)

### PR DESCRIPTION
**Pull request by brilhana**
Added `first` and `last` methods. `head` and `tail` now defaults to `n = 6`.

### CODE REVIEW

File: src/split.jl [hunk 1]
```diff
@@ -31,21 +31,38 @@ findwhen(ta::TimeArray{Bool, 1}) = ta.timestamp[find(ta.values)]
...
     ncol          = length(ta.colnames)
     tail_start = length(ta)-n+1
     new_timestamp = ta.timestamp[tail_start:end]
     new_values    = ta.values[tail_start:end, 1:ncol]
     TimeArray(new_timestamp, new_values, ta.colnames, ta.meta)
 end
 
+###### first, last ###########
+
+function first(ta::TimeArray)
```
Comment (iblis17) [inline src/split.jl]: Maybe we should overload `Base.first` and `Base.last`?

Comment (iblis17) [reply src/split.jl]: and I think this function should reuse `head`
```julia
Base.first(ta::TimeArray)  = head(ta, 1)
Base.last(ta::TimeArray)  = tail(ta, 1)
```

Comment (brilhana) [reply src/split.jl]: Didn't think about it, good idea.

File: src/TimeSeries.jl [hunk 2]
```diff
@@ -7,7 +7,8 @@ using Base.Dates
 using RecipesBase
 
 export TimeArray, AbstractTimeSeries,
-       when, from, to, findwhen, find, timestamp, values, colnames, meta, head, tail,
+       when, from, to, findwhen, find, timestamp, values, colnames, meta,
+       head, tail, first, last,
```
Comment (iblis17) [inline src/TimeSeries.jl]: So, I think we do not need to export them now.

File: test/split.jl [hunk 3]
```diff
@@ -55,52 +55,69 @@ end
...
-        @test head(ohlc).meta         == "AAPL"
-
-        @test length(tail(ohlc))      == 1
-        @test length(tail(ohlc,1))    == 1
-        @test tail(ohlc).timestamp[1] == Date(2001,12,31)
-        @test tail(ohlc).values       == [22.51  22.66  21.83  21.9]
-        @test tail(ohlc).meta         == "AAPL"
+    @testset "head, tail, first and last methods work with default n value on multi column TimeArray" begin
+        @test length(head(ohlc)) == 6
+        @test head(ohlc,1)       == [104.88  112.5  101.69  111.94]
```
[the remark this answers is not in the document]
Comment (brilhana) [reply test/split.jl]: Indeed. Fixed it now.

**Comment by brilhana** [comment]
Some tests fail, I'm fixing them now.

**Comment by iblis17** [comment]
Thanks for your contributions!

**Pull request merged**